VerbalExpressionsUnitTests/SanatizeTests.cs

```diff
@@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
     public class SanatizeTests
```

Comment:
I'm the last one to argue about formatting - obviously the code was dritten without resharper...
Greetings
Am 02.02.2016 19:22 schrieb SignpostMarv <notifications@github.com>:In VerbalExpressionsUnitTests/SanatizeTests.cs:
> @@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
>      public class SanatizeTests

Given that it required removing the now-unsupported exception attributes, that VS reported line ending issues in some files & git reported whitespace errors, I felt that having a consistent formatting on the files prior to updating nunit was preferable to performing the formatting afterwards.

—Reply to this email directly or view it on GitHub.